Documentation/getting_start.md

```diff
@@ -7,14 +7,32 @@ This section tells you how to deploy the yurt-edgex-manger to a cluster
 To deploy or test the yurt-edgex-manager alone, we can start from a generic kubernetes cluster. i.e. you can create a cluster with 3 nodes by kind. More kind information can refer [kind usage](https://kind.sigs.k8s.io/)
 Note: the kubernetes version must < v1.21.0
 ```
-kind create cluster
+vim my-cluster-multi-node.yaml
+
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+```
+If you want to deploy other versions of the image, you can refer [image version](https://github.com/kubernetes-sigs/kind/releases/tag/v0.17.0).
+
+```
+kind create cluster --config my-cluster-multi-node.yaml --name openyurt --image kindest/node:v1.20.15@sha256:a32bf55309294120616886b5338f95dd98a2f7231519c7dedcec32ba29699394
```

Comment:
Could you upgrade the kindest/node image to version 1.24? We can already support 1.24, so it's better to let users install the newer version.
The configuration of the image can be written in a yaml file:
```yaml
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
  image: kindest/node:v1.24.6@sha256:97e8d00bc37a7598a0b32d1fabd155a96355c49fa0d4d4790aab0f161bf31be1
- role: worker
  image: kindest/node:v1.24.6@sha256:97e8d00bc37a7598a0b32d1fabd155a96355c49fa0d4d4790aab0f161bf31be1
- role: worker
  image: kindest/node:v1.24.6@sha256:97e8d00bc37a7598a0b32d1fabd155a96355c49fa0d4d4790aab0f161bf31be1
```